### ISSUE TITLE
BiSeNetV2训练时的评估异常

### ISSUE THREAD

**Issue by ucsk** [PaddlePaddle/PaddleX#1146]
问题类型：模型训练  

**PaddleX版本**  
您使用的PaddleX版本
`pip install paddlex` (default 2.0.0)

**问题描述**  

二分类训练 `BiSeNetV2` 模型时，在验证集上的评估结果保持不变，即 `类别 1` 无法得到正常预测。
通过调整学习率 [0.01, 1] 等均无法改变该情况。
例如：
```
[INFO]	[EVAL] Finished, Epoch=25, miou=0.481154, category_iou=[0.962308 0.      ], oacc=0.962308,
category_acc=[0.96230811 0.        ], kappa=0.000000, category_F1-score=[0.98079203 0.        ] .
```

以下是部分代码（若使用UNet等模型则可以完成正常训练）：
```python
......

train_transforms = T.Compose([
    T.Resize(512),
    T.Normalize(mean=[0.5, 0.5, 0.5], std=[0.5, 0.5, 0.5]),
])

eval_transforms = T.Compose([
    T.Resize(512),
    T.Normalize(mean=[0.5, 0.5, 0.5], std=[0.5, 0.5, 0.5]),
])

......

model = pdx.seg.BiSeNetV2(
    num_classes=2,
    use_mixed_loss=False)

model.train(
    num_epochs=200,
    train_dataset=train_dataset,
    save_dir='output/BiSeNetV2',
    eval_dataset=eval_dataset,
    train_batch_size=25,
    learning_rate=0.1,
    log_interval_steps=100,
    pretrain_weights='CITYSCAPES',
    save_interval_epochs=5,
    use_vdl=False)
```

**Comment by ucsk** [comment]
我在PaddleX GUI 2.0.0上测试，发现起始的学习率总是固定为 `lr=0.001000`，可能这是导致评估失败的原因之一。

**Comment by will-jl944** [comment]
PaddleX分割模型学习率默认使用的是[polynomial decay](https://www.paddlepaddle.org.cn/documentation/docs/zh/api/paddle/optimizer/lr/PolynomialDecay_cn.html#polynomialdecay)策略。`model.train`中设置的`learning rate`为初始学习率，最终学习率为0。

根据你的描述，因该是训练参数设置问题，可以尝试一下方法：
1. 参考[API文档](https://github.com/PaddlePaddle/PaddleX/blob/release/2.0.0/docs/apis/models/semantic_segmentation.md#train)对每一个参数含义的解释对参数进行调整。
2. 根据你的具体使用场景以及数据特性，使用混合损失函数，详见[文档](https://github.com/PaddlePaddle/PaddleX/blob/release/2.0.0/docs/apis/models/semantic_segmentation.md#paddlexsegbisenetv2)。
3. 使用自定义优化器替换默认优化器，可参考[使用示例](https://github.com/PaddlePaddle/PaddleX/blob/7e9ec209f984404a15df5562b1543ef5affedf54/tutorials/train/image_classification/mobilenetv3_large_w_custom_optimizer.py#L38)。
